src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -119,7 +119,7 @@ fabric.Breadcrumb.prototype = (function() {
   var _addBreadcrumbItems = function(maxItems) {
     _resetList(_breadcrumbList);
     var i = _itemCollection.length - maxItems;
-    i=i<0? 0 : i;
+    i = i < 0? 0 : i;
```

Comment:
Thanks for making these changes. Nit, can you add a space between the 0 and the ?.